src/bid_order.rs

```diff
@@ -237,7 +239,7 @@ pub fn migrate_bid_orders(
     if VersionReq::parse(">=0.15.1, <0.16.2")?.matches(&current_version) {
         let existing_bid_order_ids: Vec<Vec<u8>> = bucket_read(store, NAMESPACE_ORDER_BID)
             .range(None, None, Order::Ascending)
-            .map(|kv_bid: StdResult<Pair<BidOrderV1>>| {
+            .map(|kv_bid: StdResult<Record<BidOrderV1>>| {
```

Comment:
is Pair no longer used?

Reply:
yep. See https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md#016---100-beta